isomerpages/isomercms-frontend
# refactor(layouts): split workspace layout into view and edit

## Problem



The new identity phase 2 features require a new type of layout that does not contain a sidebar, and our existing site layout is not able to cater for such a case.

## Solution



**Breaking Changes**




**Features**:

- The site layouts have been refactored and split into an edit and a view layout. The existing workspace layout is the SiteEditLayout (renamed from SiteViewLayout) whereas the new layout used by the identity phase 2 features is the new SiteViewLayout.
- The header used for both the SiteEditLayout is also preserved, whereas the header for the SiteViewLayout is a temporary placeholder that will be replaced by @alexanderleegs as part of the notifications feature.

## Before & After Screenshots

No screens are available for the new SiteViewLayout yet, will be implemented as part of the site dashboard.

## Tests




1. Run `npm run dev` on this branch.
2. Log in and navigate to any site.
3. Verify that the existing pages in the workspace still look the same.

## Deploy Notes




*None*
